statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -239,6 +239,12 @@ func (wr *WidgetsRender) prepareBlockInfo() {
 	crossCheckBlockHeight := wr.presenter.GetCrossCheckBlockHeight()
 	rows[4] = []string{fmt.Sprintf("Cross check block height: %s", crossCheckBlockHeight)}
 
+	shardId := wr.presenter.GetShardId()
+	if shardId != uint64(sharding.MetachainShardId) {
+		highestBlockInShard := wr.presenter.GetHighestFinalBlockInShard()
+		rows[4][0] += fmt.Sprintf(" ,highest block nonce in shard: %d", highestBlockInShard)
```

Comment:
add a space after the comma and also set the default value for previous metric

Reply:
done.